### DISCUSSION TITLE
SDRTrunk Metadata changing Rdio-scanner talkgroup name

### DISCUSSION THREAD

**Discussion by wmg819** [chuot/rdio-scanner#211] [General]
I was having same issue as posted in SDRTrunk Metadata Dirwatch Parser Broken in 6.4.1 #206. It seems no matter what was in the talkgroup name space which 4th row of the display screen (blank or otherwise) it was being replaced with TRAFFIC or the sdr trunk alias in the meta data and database when auto populate was on in both the options and system section. The dirwatch was set to sdrtrunk with the defaults set for the mask. The only way to stop that from happening it seemed was to keep those options off to stop the changes but this would prevent new talk groups from being logged. 

I did finally find a way to fix this by setting up an alias list in sdrtrunk in which the alias and group names were left blank with the talk group range set from 0 to 65535. Now the calls from sdrtrunk are ingested to rdio-scanner using the system info the way I have it set up in rdio-scanner and when the auto populate options are set to on new talk groups are added by thier talk group number without changing the already identified talk groups. in the rdio-scanner system talk group list.   This works in both versions 6.4.1 and 6.4.2 in window and ubuntu 18.04

**Comment by chuot** [comment]
Thanks for sharing this.

Indeed, the dirwatch type sdr-trunk only works with recordings from the aliases tab on sdr-trunk. I added a note on Rdio Scanner in that regards.
